plugins/log_full.yml

```diff
@@ -53,3 +53,13 @@ plugin:
     - $NAMESPACE
     - --context
     - $CONTEXT
+  download-container-logs:
+    shortCut: Ctrl-D
+    description: "download full log"
+    scopes:
+    - containers
+    command: bash
+    background: true
+    args:
+    - -c 
+    - "kubectl logs -n $NAMESPACE $POD -c $NAME --context $CONTEXT > $POD.log"
```

Comment:
Do we want the download to land in the cwd?

Reply:
yes it will download to current working directory, or i can change it to $HOME ?

Reply:
log files have the potential to be sizeable and possibly ephemeral, wondering if parking them in tmp might be a better solution?

Reply:
Perhaps add a config setting so the user can define where logs should be downloaded to.
$HOME seems like a reasonable default. /tmp can be very small on security hardend systems.
If size is an issue, compress it on the fly.